src/stoppingtest.jl

```diff
@@ -17,10 +28,11 @@ function test_stopping_criterion(param::SDDPparameters, stats::SDDPStat)
     lb = stats.lower_bounds[end]
     ub = stats.upper_bounds[end]
     check_gap = (abs((ub-lb)/lb) < param.gap)
-    check_iter = stats.niterations > param.maxItNumber
+    check_iter = stats.niterations >= param.maxItNumber
```

Comment:
I disagree! SDDPStats is 0-indexed, and you want to stop the algorithm in a 1-indexed manner. Please fix this issue. If user specifies n iterations to run SDDP, he does not want to compute n + 1 iterations.